azure-iot-device/tests/common/pipeline/pipeline_stage_test.py

```diff
@@ -80,8 +80,8 @@ class LocalTestObject(StageRunOpTestBase):
         @pytest.fixture(params=unknown_ops)
         def op(self, request, mocker):
             op = make_mock_op_or_event(request.param)
-            op.callback = mocker.MagicMock()
-            add_mock_method_waiter(op, "callback")
+            op.callbacks.append(mocker.MagicMock())  # TODO: make this simpler
```

Comment:
># TODO: make this simpler [](start = 53, length = 25)

please take ownership of this TODOD (# CT-TODO) and/or make it easier to understand.  I'm not sure how this particular line of code could be any simpler.